css/template-2.css

```diff
@@ -85,6 +89,39 @@
   object-fit: cover;
 }
 
+.card:hover > .card-overlay {
+  display: flex;
+}
+
+.eye,
+.link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)